### ISSUE TITLE
Android Kotlin Fundamentals codelab: missing next step

### ISSUE THREAD

**Issue by neilwithdata** [google-developer-training/android-kotlin-fundamentals-apps#593]
**Name of the Codelab or Codelab URL**
Android Kotlin Fundamentals: Lifecycles and logging
https://developer.android.com/codelabs/kotlin-android-training-lifecycles-logging

**Describe the problem**
The end of this codelab should point to the next codelab to complete in the sequence (like every other codelab in the "Android Kotlin Fundamentals" course does). I believe that the codelab in the sequence should be https://developer.android.com/codelabs/kotlin-android-training-complex-lifecycle

**In which lesson and step of the codelab can this issue be found?**
Android Kotlin Fundamentals: Lifecycles and logging - missing step 10.

**How to reproduce?**
N/A

**Versions**
1. What version of Android Studio are you using? N/A
2. What API level are you targeting? N/A

**Additional information**

**codelab:** android-kotlin-fundamentals

**Comment by azrael-42** [comment]
I have experienced this also today. Thank you @neilwithdata for giving a suggested next link - I did not know where to go next.

It would help if this statement, taken from the introduction of https://developer.android.com/codelabs/kotlin-android-training-view-model#0 was accurate.
All the course codelabs are listed on the Android Kotlin Fundamentals codelabs landing page.

The landing pages only lists the sections, not the individual codelabs that make up the sections.

**Comment by suread** [comment]
I have experienced this also today. Thank you @neilwithdata for giving a suggested next link - I did not know where to go next.

It would help if this statement, taken from the introduction of https://developer.android.com/codelabs/kotlin-android-training-view-model#0 was accurate.
All the course codelabs are listed on the Android Kotlin Fundamentals codelabs landing page.

The landing pages only lists the sections, not the individual codelabs that make up the sections.
